docs/source/installing.rst

```diff
@@ -61,7 +61,7 @@ To start the installation process using the Boards Managaer, follow these steps:
 Windows
 -------
 
-.. warning:: Arduino ESP32 core v2.x.x cannot be used on Windows 7 or earlier. The Windows 32 bits OS is no longer supported by this toolchain.
+.. warning:: Arduino ESP32 core v2.x.x cannot be used on Windows 7, Windows 8.x x86 (32 bits) or earlier. The Windows 32 bits OS is no longer supported by this toolchain.
```

Comment:
Just small note, probably make more sence to say first Windows 8.x x86 (32 bits), Windows 7 or ealier..to keep timeline of Windows releases.

Some other thing is to probably add MacOS as well? Starting from MacOS 10.15, earlier versions are not supported. (I'm not a MacOS user though :) )